collections/individual/index.php

```diff
@@ -20,6 +22,8 @@
 $clid = array_key_exists('clid', $_REQUEST) ? $indManager->sanitizeInt($_REQUEST['clid']) : 0;
 $format = isset($_GET['format']) ? $_REQUEST['format'] : '';
 
+$shouldUseMinimalMapHeader = $SHOULD_USE_MINIMAL_MAP_HEADER ?? false;
```

Comment:
I would just make this a single line of if($SHOULD_USE_MINIMAL_MAP_HEADER ?? false) include(...). I realize it is also being used for the css logic but if you look at the other comments you see that I recommend styling changes that wouldn't require this conditional spacing.

Reply:
I just tried this. I can't seem to get the header to display nicely along with the "Open Search Panel" button in collections/map/index.php, even if I explicitly set `display: block;` to the minimal header.

Maybe we meet about this and problem solve together in Part 2? I have a feeling that I have something to learn here, but I couldn't find an immediate solution.

I added it to the TODO list in Part 2: https://github.com/BioKIC/Symbiota/issues/1428.